docs/docs/overview/why_secretless.md

```diff
@@ -6,32 +6,36 @@ description: Secretless Broker Documentation
-
-Exposing plaintext secrets to clients (both users and machines) is hazardous from both a security and operational standpoint. First, by providing a secret to a client, the client becomes part of the threat surface. If the client is compromised, then the attacker has a good chance of obtaining the plaintext secrets and being able to establish direct connections to backend resources. To mitigate the severity of this problem, important secrets are (or should be) rotated (changed) on a regular basis. However, rotation introduces the operational problem of keeping applications up to date with changing passwords. This is a significant problem as many applications only read secrets on startup and are not prepared to handle changing passwords.
+Keeping secrets in a vault is a good practice. However, when a client “checks out” a secret from a vault, where does it go? Often, directly into unprotected application memory at which point that client app has now part of the threat surface. There are a stringent set of best practices and recommendations that should be be followed by every user and application which checks secrets out of a vault, but even if you harden the app in every possible way it does not protect you from 0-day vulnerabilities in the app, underlying framework, and/or the programming language used.
 
 It can be a losing battle to try and train every user and application developer about how to safely handle a secret once they’ve obtained it even if you can mitigate all of the attack surfaces of the client app itself. There are so many things that can go wrong, and people (and applications) have better things to worry about than babysitting secrets (such as doing their jobs). Do you want developers to write features or try to be security engineers too?
 
-The Secretless Broker is specifically written to be really good at interacting with a variety of vaults and abstracting away the security to a specialized module that can easily be audited and expanded with custom plugins. It does a really good job of securing any secrets that it might obtain from a vault, and it does it the same way no matter the developer, technology, or platform. Additionally, it knows what to do when a secret has been changed so that the client users and code never have to respond to changing secrets or even know that they are changing.
+The Secretless Broker is specifically written to be really good at interacting with a variety of vaults and abstracting away the security to a specialized module that can easily be audited and expanded with custom plugins. It does a really good job of securing any secrets that it might obtain from a vault, and it does it the same way no matter the developer, technology, or platform.
 
 In summary, when the client connects to the target service through the Secretless Broker:
 
+- **The client does not have to know how to fetch credentials**
```

Comment:
s/`know how to code its APIs`/`know how to interact with it`

Reply:
:/ but just before that, it says "it doesn't have to know how to interact with a secrets vault"

Reply:
Hmm.. you're right... maybe just join the sentences? "The client/app no longer has to directly interact with a secrets vault and no code needs to be changed to fetch credentials before opening a connection."